### DISCUSSION TITLE
Can add new extension

### DISCUSSION THREAD

**Discussion by xXCats** [aniyomiorg/aniyomi#755] [Ideas]
you can add the Italian extension of the site
AnimeUnity

URL:https://www.animeunity.tv/[](https://www.animeunity.tv/)

**Comment by Xujiin** [comment]
Animedao will be a good new extension

**Comment by swtstvn** [comment]
https://nekopoi.care/ will be good hentai extension :D

**Comment by Xujiin** [comment]
chia-anime.tv is a good extension to be added
